### PR TITLE
Add more command information to `VSCode.md`

### PR DESCRIPTION
**Pull request by lindsayad**
Now is the best time for me to write this down as I'm rewiring my brain

### CODE REVIEW

File: modules/doc/content/help/development/VSCode.md [hunk 1]
```diff
@@ -33,13 +33,36 @@ well as the installed extensions.
...
+- +`Option+Repeated Clicks`+ Place multiple cursors (one per each click)
+- +`Cmd+S`+ Save current file
+- +`` Ctrl+` ``+ Open terminal
+- +`Cmd+\`+ Split screen vertically
+- +`Cmd+W`+ Close current file. If last file in split, then split will disappear
+- +`Ctrl+G`+ Goto line
+
+### Focuses
+
+- +`findInputFocussed`+ Cursor is in the Find box (note that cursor in the Replace box of the Find and Replace box *does not count*)
```
Comment (loganharbour) [inline modules/doc/content/help/development/VSCode.md:63]:
```suggestion
- +`findInputFocused`+ Cursor is in the Find box (note that cursor in the Replace box of the Find and Replace box *does not count*)
```

File: modules/doc/content/help/development/VSCode.md [hunk 2]
```diff
@@ -33,13 +33,36 @@ well as the installed extensions.
...
   issued the `code .` command. No need to know the precise path or even the precise spelling of the
   filename!
+- +`Cmd+[`+ Decrease indent (Outdent)
+- +`Cmd+]`+ Increase indent (Indent)
+- +`Cmd+F`+ Find
+- +`Cmd+Option+F`+ Find and replace
+  - +`Enter`+ Find next match without replacement
+  - +`Ctrl+Enter`+ Replace
+- +`Cmd+Option+Left/Right Arrow`+ tab between open editor files as ordered on the screen
+- +`Ctrl+Tab`+ tab between editor files in the order they were opened
```
Comment (loganharbour) [inline modules/doc/content/help/development/VSCode.md:52]:
```suggestion
- +`Cmd+Option+Left/Right Arrow`+ Tab between open editor files as ordered on the screen
- +`Ctrl+Tab`+ Tab between editor files in the order they were opened
```

**Review by loganharbour — APPROVED** [review]
Beautiful.

I should add stuff for rod on here

File: modules/doc/content/help/development/VSCode.md (hunk 1 above)
Comment (lindsayad) [reply modules/doc/content/help/development/VSCode.md:63]: You'll need to take up this spelling mistake with the vs code devs 😆 

Screenshot 2024-12-05 at 4 54 01 PM

Comment (lindsayad) [reply modules/doc/content/help/development/VSCode.md:63]: You'll need to take up this spelling mistake with the vs code devs 😆 

Screenshot 2024-12-12 at 10 42 56 AM

**Review by dschwen — APPROVED** [review]

**Pull request merged**